openapiart/common.py

```diff
@@ -903,11 +914,17 @@ def _validate_types(self, property_name, property_value):
             msg = "property {} shall be of type {} at {}".format(
                 property_name, details["type"], self.__class__
             )
+
+            itemtype = (
```

Comment:
without this change, were we seeing any bugs ?

Reply:
yes the the validation was not happening for list of type integers